src/APICalls/APICalls.js

```diff
@@ -20,4 +20,22 @@ export const getAllCourses = () => {
     })
 }
 
+export const postEvent = (courseId, date, teeTime, openSpots, numHoles, isPrivate, hostId, selectedFriends) => {
+  return fetch('http://d48fc90c7809.ngrok.io/api/v1/events', {
```

Comment:
Might want a way to make this dynamic so that we can easily switch to production when needed.